src/legacy/core_plugins/kibana/public/visualize/np_ready/editor/editor.js

```diff
@@ -405,6 +405,9 @@ function VisualizeAppController($scope, $route, $injector, $timeout, kbnUrlState
       stateContainer
     );
     vis.uiState = persistedState;
+    vis.uiState.on('reload', () => {
```

Comment:
This is making sure the reload is picked up when visualize embeddable is used in visualize editor